docker/Makefile

```diff
@@ -14,12 +14,11 @@
 
 all: push
 
-TAG ?= 0.1
+TAG ?= latest
```

Comment:
Wouldn't it be better to get the version from gradle?

Reply:
Possible, but in a Makefile this can be passed through an environment variable.
`TAG=${VERSION} make`

It is better to do this in CI than in a Makefile.